android/src/main/java/io/parity/signer/domain/Navigation.kt

```diff
@@ -30,82 +30,6 @@ interface Navigator {
 }
 
 
-/**
- * Class to navigate within rust state-machine area. It is one (big) part of compose-based navigation.
- * This class have nothing to do with composa-based navigation.
- */
-class SignerNavigator() : Navigator {
-
-	private val appContext: Context get() = ServiceLocator.appContext
-	private val activity get() = ServiceLocator.activityScope?.activity
-
-	private val _actionResult = MutableStateFlow<ActionResult?>(
-		null
-	)
-	val actionResult: StateFlow<ActionResult?> = _actionResult.asStateFlow()
-
-	private suspend fun navigateRust(
```

Comment:
This class is used to change general application state. It is not used anymore anywhere. Since last PR. Transaction using navigation calls, but not from SignerNavigator.